_kbarticles/2025-04-07-opensearch-alerting.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
```

Comment:
Can we make this a news post instead of a knowledge base article? The content can be roughly the same, but I want customers to understand that this is a new feature we're releasing